src/hello-n.py

```diff
@@ -4,3 +4,12 @@
 n = int(input("How many times should I loop? "))
 
 # Now, print "Hello, World!" n times.
+
+for number in range(0, n):
```

Comment:
You could just use `range(n)` here. When you start from zero and go to `n` in steps of one, the short version of `range()` works.